### ISSUE TITLE
adblocking

### ISSUE THREAD

**Issue by Venipa** [Venipa/ytmdesktop2#19]
### tldr; i wont implement adblocking for now
<a href="https://adguard.com/?aid=32949"><img src="https://cdn.adguard.info/website/adguard.com/affiliate/dashboard/images/AdGuard/en/728x90.png" srcset="https://cdn.adguard.info/website/adguard.com/affiliate/dashboard/images/AdGuard/en/728x90@2x.png 2x" width="728" height="90"></a>

Since adblocking wont be a official feature to be implemented in my desktop app i can only recommend AdGuard to block most of the ads on youtube, youtube music and other websites.
Try AdGuard: https://adguard.com/?aid=32949
or try AdGuard VPN: https://adguard-vpn.com/?aid=32949

✅ Blocks all kinds of ads and trackers
✅ Removes annoying web elements
✅ Saves traffic and speeds up page loading
✅ Works for both browsers and apps
✅ Maintains site functionality and appearance
... for more information visit the links above

You'll support me if you buy from that link.

Trustpilot reviews: https://www.trustpilot.com/review/adguard.com

**Comment by senolfndk** [comment]
Anyway, take it easy, I'm uninstalling the app

**Comment by Jonybtw** [comment]
Hi, won't it be easy to implement the uBlock to the app as an extension since (I assume) it's a browser?

**Comment by Venipa** [comment]
adblocking will be supported unofficially by using plugin system, it will take time tho until its available as plugin

**Comment by Walkoud** [comment]
waiting ....

**Comment by erhszo** [comment]
waiting...........

**Comment by Jonybtw** [comment]
waiting.....................

**Comment by tusharchopra07** [comment]
waiting...................

**Comment by az963258** [comment]
waiting...........................

**Comment by Salah242222** [comment]
waiting...........................

**Comment by TheWeirdHomosapien** [comment]
waiting............................

**Comment by Salah242222** [comment]
waiting............................

**Comment by Venipa** [comment]
guys this issue is only to exist to remind me that this is a possibility for a feature but its not on my priority list
![601052264123203605](https://github.com/Venipa/ytmdesktop2/assets/17952364/f93cb89a-93e9-409d-8485-fa79dc797b47)

**Comment by alexonpeace** [comment]
You don't have to add ad blocking, how about that plugin system

**Comment by ArchB1W** [comment]
This is a solo project the developer does not owe you built-in adblock or a plugin system, spamming "waiting..." is not productive nor helpful, the developer'll get around to whatever they want whenever they want, if they even want to at all, remember that developers have no obligation to do anything you ask of them. Also remember there is a difference between requesting a feature and pestering someone with spam.

**Comment by TheWeirdHomosapien** [comment]
> This is a solo project the developer does not owe you built-in adblock or a plugin system, spamming "waiting..." is not productive nor helpful, the developer'll get around to whatever they want whenever they want, if they even want to at all, remember that developers have no obligation to do anything you ask of them. Also remember there is a difference between requesting a feature and pestering someone with spam.

I apologize, I just wanted my "waiting..." to be another sign of support for the feature. I can see why it is annoying or rude. I will be more mindful.

**Comment by Natanel-Shitrit** [comment]
I've added ad-skipping plugin in [my fork](https://github.com/Natanel-Shitrit/ytmdesktop2), If you are interested I would happily PR it for everyone to use.

**Comment by Walkoud** [comment]
i'm still waiting....

**Comment by HitogiDev** [comment]
I see something was merged but I don't see any option to activate the ad skipper

**Comment by UnTamed-Fury** [comment]
i am waiting aswell
